### PR TITLE
Updated composer.json and DatabaseAnalyzer.php

### PR DESCRIPTION
**Pull request by theStormWinter**
Updated for using nette/php-generator up to version 3.2.3 
Repaired in DatabazeAnalyzer.php array key "Extra" to lowercase

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
   "require": {
     "php": ">=7.1",
     "nette/database": "^2.4.0",
-    "nette/php-generator": "^2.5|3.1",
+    "nette/php-generator": "^2.5|3.2.3",
```
Comment (mabar) [inline composer.json]: `^2.5.0|^3.0.0`, exact 3.x version is a bug

File: src/Analyser/Database/DatabaseAnalyser.php [hunk 2]
```diff
@@ -70,7 +70,7 @@ protected function analyseColumns(Table $table): void
 			$column->setNullable($col['nullable']);
 			$column->setType(Helpers::columnType($col['nativetype']));
 			$column->setDefault($col['default']);
-			$column->setOnUpdate(Strings::contains($col['vendor']['Extra'], 'on update'));
+			$column->setOnUpdate(Strings::contains($col['vendor']['extra'], 'on update'));
```
Comment (mabar) [inline src/Analyser/Database/DatabaseAnalyser.php]: Capitalized key was introduced by #16, someone should check proper behavior before any changes of that https://github.com/contributte/nextras-orm-generator/commit/b0002a1da7578586a83576d341e224567eaf79be

**Comment by f3l1x** [comment]
ping @theStormWinter

**Comment by f3l1x** [comment]
Merged in #29

**Pull request closed without merging**